### PR TITLE
Enforce no consecutive blank lines with rubocop (Layout/EmptyLines)

### PR DESCRIPTION
**Pull request by zzak**
This applies the rules for #47848 programmatically.

Autocorrect all rubocop offenses for [Layout/EmptyLines](https://docs.rubocop.org/rubocop/cops_layout.html#layoutemptylines)

```
3156 files inspected, 53 offenses detected, 53 offenses corrected
```

See the diff for the full changes.

### CODE REVIEW

**Comment by yahonda** [comment]
https://github.com/rails/rails/pull/40193#issuecomment-688581065 says at least one of the double-blank lines is intentional.

File: activestorage/app/models/active_storage/preview.rb [hunk 1]
```diff
@@ -107,7 +107,6 @@ def variant
       image.variant(variation).processed
     end
 
-
```
Comment (zzak) [inline activestorage/app/models/active_storage/preview.rb:110]: Intentional:

https://github.com/rails/rails/pull/30667/files#r140381739

**Comment by zzak** [comment]
It seems like there are multiple places where the carriage returns are intentional, without going through each one I think we should allow at least 2 empty lines, but not sure we can enforce that with rubocop yet.

**Comment by zzak** [comment]
My other idea was to convert the intentional CRs to code comments, since that is way more explicit and future code readers will be able to understand why it's there. Arbitrary new lines seems like not the most accessible way to convey intent IMO.

However, as I just pointed out, there seem to be multiple places so figuring out exactly which ones are intentional is a guessing game. 🤔

**Comment by zzak** [comment]
Without rubocop allowing us to easily configure the offset of empty lines (default is 2, but we could say 3), or knowing exactly which newlines are intentional, this is going to be difficult to resolve. Closing.

**Pull request closed without merging**